scripts/build-packages.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path");
+const fs = require("fs");
+const shell = require("shelljs");
+
+shell.cd(path.join(__dirname, ".."));
+
+const packages = fs
+    .readdirSync(path.join(__dirname, "..", "packages"))
+    .filter(name => !name.startsWith("."));
+
+for (const package of packages) {
+    const pkgPath = path.join(__dirname, "..", "packages", package);
+    const pkgJSONPath = path.join(pkgPath, "package.json");
+
+    if (fs.existsSync(pkgJSONPath)) {
+        const pkgJSON = require(pkgJSONPath);
+
+        if (pkgJSON && pkgJSON.scripts && pkgJSON.scripts.build) {
+            console.log(`\nbuilding "/packages/${package}"\n`);
+
+            if (shell.exec(`yarn --cwd "${pkgPath}" build`).code !== 0) {
```

Comment:
If the command  failed, will the terminal print out the complete error message and stack?

Reply:
Yes